### PR TITLE
Clarify docs around x-pack monitoring

### PR DESCRIPTION
**Pull request by ruflin**
When the Elasticsearch output is used, no additional hosts can be configured in the X-pack configs.

### CODE REVIEW

File: libbeat/docs/monitoring/monitoring-beats.asciidoc [hunk 1]
```diff
@@ -46,7 +46,8 @@ xpack.monitoring:
   elasticsearch:
 --------------------
 
-Otherwise, you must specify additional configuration options. For example:
+If you are using a different output then Elasticsearch, you must specify
```
Comment (dedemorton) [inline libbeat/docs/monitoring/monitoring-beats.asciidoc]: I thinks these details are still a little hard to follow. Are you saying that users *must* send monitoring data to the production cluster? If so, you should also modify the sentence that begins on line 37 (remove "and you want to use the same {es} production cluster and credentials"). So I would change this to say something like:

```
. Add the `xpack.monitoring` settings in the {beatname_uc} configuration file. If you configured {es} output,
specify the following minimal configuration:
.
.
.
If you configured a different output, such as {ls}, you must specify additional configuration options.
For example:
.
.
.
NOTE: Currently you must send monitoring data to the same cluster as all other events. If you configured the
{es} output, do not specify additional hosts in the monitoring configuration. 
```

Comment (ruflin) [reply libbeat/docs/monitoring/monitoring-beats.asciidoc]: Thanks for the proposal. Adjusted accordingly.

File: libbeat/docs/monitoring/monitoring-beats.asciidoc [hunk 2]
```diff
@@ -33,20 +33,17 @@ information, see
...
-options:
+. Add the `xpack.monitoring` settings in the {beatname_uc} configuration file. If you
+configured {es} output, specify the following minimal configuration:
 +
 --
 [source, yml]
 --------------------
-xpack.monitoring:
-  enabled: true
-  elasticsearch:
```
Comment (ruflin) [inline libbeat/docs/monitoring/monitoring-beats.asciidoc]: @urso Any reason the `elasticsearch` part was in before?
